Kudu.Core/Environment.cs

```diff
@@ -101,6 +104,7 @@ public class Environment : IEnvironment
             _dataPath = Path.Combine(rootPath, Constants.DataPath);
             _jobsDataPath = Path.Combine(_dataPath, Constants.JobsPath);
             _jobsBinariesPath = Path.Combine(_webRootPath, Constants.AppDataPath, Constants.JobsPath);
+            _alwaysOnEnabled = IsAlwaysOnEnabled();
```

Comment:
Ok. There isn't a central place where we manage environmental settings, etc.? That's what this looked like to me.